# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/rpcs3/hotfix-noglx.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- rpcs3/3rdparty/CMakeLists.txt.vanilla	2024-02-23 23:36:55.515000000 +0000
+++ rpcs3/3rdparty/CMakeLists.txt	2024-02-23 23:40:08.990000000 +0000
@@ -119,7 +119,7 @@
 elseif(APPLE)
 	target_link_libraries(3rdparty_opengl INTERFACE OpenGL::GL OpenGL::GLU)
 else()
-	target_link_libraries(3rdparty_opengl INTERFACE OpenGL::GL OpenGL::GLU OpenGL::GLX)
+	target_link_libraries(3rdparty_opengl INTERFACE OpenGL::GL OpenGL::GLU)
 endif()
 
 
